sphericalmercator/bq/doc/REFERENCE.md

```diff
@@ -0,0 +1,28 @@
+## Reference
+
+### SPHERICALMERCATOR
+
+#### BBOX
+
+{{% bannerNote type="code" %}}
+sphericalmercator.(x NUMERIC, y NUMERIC, zoom NUMERIC, tileSize NUMERIC )
+{{%/ bannerNote %}}
+
+Returns bbox array of values in form [w, s, e, n].
+
+* `x`: `NUMERIC` x (longitude) number.
+* `y`: `NUMERIC` y (latitude) number.
+* `zoom`: `NUMERIC` zoom level.
+* `tileSize`: `NUMERIC` size of the tiles we want to use.
+
+#### XYZ
```

Comment:
Are we using this (or the other function). How is this different from quadkey or what you can achieve with quadkeys? Looks pretty similar to me.